code/modules/core_implant/cruciform/rituals/path.dm

```diff
@@ -250,6 +250,44 @@
 	phrase = "Dominus, ait, in cujus conspectu ambulo, mittet angelum suum tecum, et diriget viam tuam." //"He replied, ‘The Lord, before whom I have walked faithfully, will send his angel with you and make your journey a success.'"
 	stats_to_boost = list(STAT_ROB = 15, STAT_TGH = 15, STAT_VIG = 15)
 
+/datum/ritual/cruciform/lemniscate/holy_boost
+	name = "Bolster the Fellowship"
+	phrase = "Benedicat tibi Dominus, et custodiat te. Ostendat Dominus faciem suam tibi, et misereatur tui. Convertat Dominus vultum suum ad te, et det tibi pacem." //“The Lord bless you and keep you; the Lord make his face shine on you and be gracious to you; the Lord turn his face toward you and give you peace.”
+	var/stats_to_boost = list(STAT_MEC = 15, STAT_COG = 15, STAT_BIO = 15, STAT_ROB = 15, STAT_TGH = 15, STAT_VIG = 15)
```

Comment:
Why so many stats

Reply:
Because... that's the whole point? The other Lemniscate litanies boost half stats on everyone who can hear. This litany boosts all stats but only on disciples, so you can't use it as effectively for things like bolstering a gate defense or aiding a medical team.

Reply:
Prob shouldnt be 15+ of every stat thats swing of 90 stats 
Maybe have each one be 10 or so

Reply:
Fair enough. If it's not impactful enough it can always be buffed later

Reply:
Sounds good